examples/pytorch/rgcn-hetero/utils.py

```diff
@@ -0,0 +1,223 @@
+"""
```

Comment:
Is this file deprecated? It seems all the utils are not used anywhere.

Reply:
They are used in link prediction, which I haven't got time to implement. I will delete it.